database/UserSchema.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema
+const DashboardSchema = require('./DashboardSchema')
+
+const UserSchema = mongoose.Schema ({
+  username: String,
+  dashboards: [{type: Schema.Types.ObjectId, ref:'Dashboard'}]
```

Comment:
`dashboards` are not a property of user - the relationship should be stored in the `Dashboard` object, unless this provides us with some needed functionality?